CONTRIBUTING.md

```diff
@@ -60,14 +61,13 @@ Guidelines for bug reports:
 
 1. If your issue is with a particular video.js plugin or subproject, please open an issue against that project. See [list of some potential other projects above](#other-repositories-where-issues-could-be-filed)
 
-1. Use the [GitHub issue search](https://github.com/videojs/video.js/issues) &mdash; check if the issue has already been reported.
-
-1. Check if the issue has already been fixed &mdash; try to reproduce it using the latest `master` branch in the repository.
+2. Use the [GitHub issue search](https://github.com/videojs/video.js/issues) — check if the issue has already been reported.
```

Comment:
Using 1. for everything makes maintain the doc a lot easier because if you add items or move stuff around you don't need to re-do the entire list.